.github/workflows/ci.yml

```diff
@@ -1,58 +1,47 @@
-name: CI
+name: Test and release when tagged
 
-on: [push, pull_request]
+on: [push]
 
 jobs:
+  lint:
+    runs-on: ubuntu-latest
```

Comment:
Might be a good idea to use `os` in the matrix so that we define everything in one job.

Reply:
Agreed, but My I will use this commit to migrate as is, and then use future commits to do this.